src/functions/keycloak/list-users/handler.ts

```diff
@@ -1,22 +1,10 @@
 import { middyfy } from "@libs/lambda";
 import type { APIGatewayProxyHandler } from "aws-lambda";
-import {
-  CreateKeycloakApiService,
-  type KeycloakApiService,
-} from "src/database/services/keycloak-api-service";
-
+import { CreateKeycloakApiService } from "src/database/services/keycloak-api-service";
+import type { User } from "src/types/keycloak/user";
 /**
- * Response schema for lambda
+ * FIXME: At this moment in KeyCloak its called: severa-user-id (string);
```

Comment:
Same as in previous file, lets leave a single comment explaining this, perhaps in the User type definition